l10n_us_product_stock/views/product_template_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_template_property_form" model="ir.ui.view">
+        <field name="name">product.template.weight.oz</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.view_template_property_form"/>
```

Comment:
@lasley Your view is not used. It should inherit from `stock.view_template_property_form`. That's where `group[@name='group_lots_and_weight']` is defined.